app/views/errors/show.html.erb

```diff
@@ -1,14 +1,31 @@
 <% set_meta_tags(title: t(".#{@rescue_response}.title")) %>
 
-<%= heading_tag t(".#{@rescue_response}.heading", default: t('.heading')) %>
+<%= heading_tag t(".#{@rescue_response}.heading", default: t('.heading')), class: "error-page__headline" %>
 
-<p class="error-page__text"><%= t(".#{@rescue_response}.intro", default: t(".#{@rescue_response}.title")) %></p>
+<p class="error-page__text<% if @rescue_response != :not_found %> error-page__text--only<% end %>"><%= t(".#{@rescue_response}.intro", default: t(".#{@rescue_response}.title")) %></p>
 
 <% if @rescue_response == :not_found %>
-  <p class="error-page__directory-text"><%= t('.not_found.message') %></p>
+  <p class="error-page__search-text"><%= t('.not_found.message') %></p>
+
+  <form data-dough-component="ClearInput" action="/en/search" method="get" role="search" class="error-page__search search">
+    <label class="visually-hidden" for="search"><%= t('search_box.label') %></label>
+    <input data-dough-clear-input class="search__input" id="search" required type="search" name="query" autocomplete="off" placeholder="<%= t('.not_found.search_placeholder') %>" />
+
+    <button data-dough-clear-input-button type="reset" class="search__clear">
+      <span class="icon icon--clear"></span>
+      <span class="visually-hidden"><%= t('search_box.clear_hidden_text') %></span>
+    </button>
+
+    <button class="search__submit" type="submit">
+      <span class="icon icon--search"></span>
+      <span class="visually-hidden"><%= t('search_box.button_text') %></span>
+    </button>
+  </form>
 
   <%= content_for(:directory) do %>
+    <%= heading_tag(level: heading_level ||= 2, class: 'l-home-directory__title') do %>
```

Comment:
I actually copied this from the home template - believe its a convention to allow you to override the h1/2/3/4/5 so it's suitable for the page its used on.